### PR TITLE
[RA2 Ch5]: Nile proofreading changes

### LINKED ISSUE #3270: [RA2 Ch5] Nile Proofreading

### PR DESCRIPTION
**Pull request by CsatariGergely**
Fixes #3270

### CODE REVIEW

**Review by rgstori — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 1]
```diff
@@ -41,26 +38,26 @@ come into picture that govern the security. Each of these layers needs to be
...
-   software needed to run an application. Before loading them to container
+   software needed to run an application. Before loading them to a container
    registry, they need to be secured by performing various checks like
    vulnerability analysis, scans etc. These images should also be signed from
    trusted sources.
 -  **Containers**: A lightweight and portable executable image that contains
    software and all of its dependencies. The containers need to be prevented from
    accessing the underlying OS like loading of kernel modules, mounting of
-   directories of underlying OS etc and ensuring that they don't run in
+   directories of underlying OS etc and it must be ensured that containers don't run in
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:51]: tense has changed from what precedes.  The new changes read:
**Containers**: A lightweight and portable executable image that contains software and all of its dependencies. The containers need to be prevented from accessing the underlying OS like loading of kernel modules, mounting of directories of underlying OS etc and it must be ensured that containers don't run in privileged mode.

Suggest
**Containers**: A lightweight and portable executable image that contains software and all its dependencies. The containers need to be prevented from accessing the underlying OS capabilities like loading of kernel modules, mounting of directories of the underlying OS, etc., and running in privileged mode.

```suggestion
   directories of the underlying OS, etc., and running in
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 2]
```diff
@@ -41,26 +38,26 @@ come into picture that govern the security. Each of these layers needs to be
...
+   container images. The access to container registry needs to be secured in
+   order to prevent unauthorised access or image tampering.
 -  **Container Images**: Stored instance of a container that holds a set of
-   software needed to run an application. Before loading them to container
+   software needed to run an application. Before loading them to a container
    registry, they need to be secured by performing various checks like
    vulnerability analysis, scans etc. These images should also be signed from
    trusted sources.
 -  **Containers**: A lightweight and portable executable image that contains
    software and all of its dependencies. The containers need to be prevented from
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:49]:
```suggestion
   software and all of the dependencies. The containers need to be prevented from
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 3]
```diff
@@ -41,26 +38,26 @@ come into picture that govern the security. Each of these layers needs to be
...
    accessing the underlying OS like loading of kernel modules, mounting of
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:50]:
```suggestion
   accessing the underlying OS capabilities like loading of kernel modules, mounting of
```

**Review by steelescotr — APPROVED** [review]
Approve these changes with consideration for edits suggested by @pgoyal01

**Pull request merged**